app/views/uploads/new.html.erb

```diff
@@ -169,8 +169,8 @@ ready = function() {
       enabled.push("cat1_r3", "cat3_r1", "pqrs_mu_individual", "pqrs_mu_group")
       disabled.push("cat1_r31", "cat3_r2");
     } else if (year == "2017") {
-      enabled.push("cat1_r31", "cat3_r2");
-      disabled.push("cat1_r3", "cat3_r1", "pqrs_mu_individual", "pqrs_mu_group")
+      enabled.push("cat1_r31", "cat3_r2", "pqrs_mu_individual", "pqrs_mu_group");

```

Comment:
Should there be a semicolon here? I'm not actually sure it matters, because Ruby, but still.